ons-spark/ancillary-topics/pydoop.ipynb

```diff
@@ -0,0 +1,438 @@
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>IncidentNumber</th>\n",
+       "      <th>DateTimeOfCall</th>\n",
+       "      <th>CalYear</th>\n",
+       "      <th>FinYear</th>\n",
+       "      <th>TypeOfIncident</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>139091</td>\n",
+       "      <td>01/01/2009 03:01</td>\n",
+       "      <td>2009</td>\n",
+       "      <td>2008/09</td>\n",
+       "      <td>Special Service</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>275091</td>\n",
+       "      <td>01/01/2009 08:51</td>\n",
+       "      <td>2009</td>\n",
+       "      <td>2008/09</td>\n",
+       "      <td>Special Service</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2075091</td>\n",
+       "      <td>04/01/2009 10:07</td>\n",
+       "      <td>2009</td>\n",
+       "      <td>2008/09</td>\n",
+       "      <td>Special Service</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>2872091</td>\n",
+       "      <td>05/01/2009 12:27</td>\n",
+       "      <td>2009</td>\n",
+       "      <td>2008/09</td>\n",
+       "      <td>Special Service</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>3553091</td>\n",
+       "      <td>06/01/2009 15:23</td>\n",
+       "      <td>2009</td>\n",
+       "      <td>2008/09</td>\n",
+       "      <td>Special Service</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  IncidentNumber    DateTimeOfCall  CalYear  FinYear   TypeOfIncident\n",
+       "0         139091  01/01/2009 03:01     2009  2008/09  Special Service\n",
+       "1         275091  01/01/2009 08:51     2009  2008/09  Special Service\n",
+       "2        2075091  04/01/2009 10:07     2009  2008/09  Special Service\n",
+       "3        2872091  05/01/2009 12:27     2009  2008/09  Special Service\n",
+       "4        3553091  06/01/2009 15:23     2009  2008/09  Special Service"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pandas_df.iloc[:, :5].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get count of rows and columns with [`.shape()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.shape.html); as we are in pandas not Spark the concept of lazy evaluation does not apply:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(5898, 26)"
```

Comment:
not sure if something is broken here, but when viewing the file as a notebook on GitHub the outputs from this point on aren't displayed

Reply:
They're in the raw code and when checking out the branch, so I'm not too concerned for now.